### ISSUE TITLE
How about transfer ownership?

### ISSUE THREAD

**Issue by y-ken** [ryotarai/fluent-plugin-cloudwatch-logs#69]
Hi  @ryotarai @cosmo0920 @okkez
It is the only plugin to send metrics to cloud watch log.
I think this plugin need to maintenance. Because some user give up the configuration.
http://qiita.com/hojishi/items/c4a2c7305ac0a5fcfd81

So, I'm afraid how about migrate maintainer to nursery team?
If you think it is hard to take a time for this plugin, some member help you.
https://github.com/fluent-plugins-nursery

instruction is here.
https://github.com/fluent-plugins-nursery/contact

**Comment by y-ken** [comment]
Due to the stopping update, It comes forked plugin.
It does not happy for the Fluentd community.
https://github.com/ryotarai/fluent-plugin-cloudwatch-logs/pull/67

**Comment by sampointer** [comment]
https://github.com/ryotarai/fluent-plugin-cloudwatch-logs/pull/67 is not a fork: it is a rewrite. 
https://github.com/sampointer/fluent-plugin-cloudwatch-ingest

**Comment by y-ken** [comment]
@sampointer 
Hi. I got it the difference.
Does fluent-plugin-cloudwatch-ingest coming to the best choise for now as cloudwatch input plugin ?

**Comment by cosmo0920** [comment]
I'm not against to @y-ken 's suggestion.
But, ideally, @fluent-plugins-nursery assumed that plugin authors want to transfer theirs plugin  ownership spontaneously.

**Comment by ryotarai** [comment]
If you would like to maintain this plugin, I can add you to collaborators of this repository. I've never heard fluent-plugins-nursery, what is that?

**Comment by cosmo0920** [comment]
> I've never heard fluent-plugins-nursery, what is that?

This is our organization which intends to maintain Fluentd plugins.
In more detail, please refer to the article: http://www.clear-code.com/blog/2016/11/9.html

**Comment by kenden** [comment]
@sampointer Why a rewrite instead of a fork? Now we have 2 solutions, it's hard to know which one is best.

**Comment by sampointer** [comment]
@kenden A couple of reasons.

Firstly, I've had very little success in getting anything merged back to this plugin (I did contribute the IAM work and some other minor fixes, but still ended up running a custom build with some of the more glaring PRs merged in). Two forks of the same plugin, with the same name, divergent, would be more confusing than a completely new plugin.

Secondly, I did start by refactoring this plugin to add my new features and whatnot. The code was hard to refactor and, being quite honest, there wasn't much left by the time I had gotten a good way in. What was left was a compromise.

I was on a commercial deadline. Ultimately it was easier for me to start and create a new plugin. I chose to share that with whomever wishes to use it.
